python/tvm/relay/frontend/pytorch.py

```diff
@@ -337,6 +337,55 @@ def _impl(inputs, input_types):
         return _op.transform.repeat(data, repeats=repeats, axis=axis)
     return _impl
 
+
+def _parse_input_data(inp):
+    import torch
+    if isinstance(inp, _expr.Var):
```

Comment:
```
    if isinstance(inp, (_expr.Var, _expr.Call):
        data = inp

```

Reply:
if one of the input is `torch.ones,`  it will first convert to `_expr.Call `for `torch.ones`, for these kind of inputs it will happen. actually we only need to differentiate `torch.Tensor` as tvm ops cannot input `torch.tensor`. 
Any tvm expr as input, tvm will handle. 
Whereever having check for `_expr.var` & `torch.tensor` is there, need to handle `_expr.call` also.

Reply:
![image](https://user-images.githubusercontent.com/15828974/79860322-153e1000-83f0-11ea-982f-e0f4b561a23b.png)

Reply:
I mean the body of Var and Call case is the same, `data = inp`, you don't need two if blocks